tonic_validate/metrics/regex_metric.py

```diff
@@ -9,11 +9,10 @@
 
 
 class RegexMetric(BinaryMetric):
-    """Searches response matching regex."""
-
     def __init__(self, name: str, pattern: str, match_count: int = 1):
         """
-        Create a metric that checks if the answer matches a given regex pattern.
+        Create a binary metric that checks if the answer matches a given regex pattern.
+        Returns 1 (True) if the pattern matches the LLM response, 0 (False) otherwise.
```

Comment:
Returns 1 (True) if the LLM response matches the pattern. Returns 0 (False) if it does not match the pattern.